low-level/sensors/impl/platforms/realstor/realstor_logical_volume_sensor.py

```diff
@@ -163,11 +184,19 @@ def run(self):
         self._read_my_msgQ_noWait()
 
         disk_groups = None
+        logical_volumes = None
+
         try:
             disk_groups = self._get_disk_groups()
 
             if disk_groups:
-                self._get_msgs_for_faulty_disk_groups(disk_groups)
+                # TODO - Disabling disk group level alerts in R1
+                # self._get_msgs_for_faulty_disk_groups(disk_groups)
+                for disk_group in disk_groups:
+                    pool_serial_number = disk_group["pool-serial-number"]
```

Comment:
all this mapping from disk group to volumes not needed anymore. Seggregate disk group code there and we will not invoke that in LDR R1, keeping for LDR R2. add code comments

Reply:
kept this because we need "disk_group" to add some info in the alert